internal/indexer/definitions/milkie.yaml

```diff
@@ -0,0 +1,58 @@
+---
+#id: Milkie
+name: Milkie
+identifier: milkie
+description: Milkie is a private torrent tracker for GENERAL.
+language: en-us
+urls:
+  - https://www.milkie.cc
+privacy: private
+protocol: torrent
+supports:
+  - irc
+  - rss
+source: custom
+settings:
+  - name: apikey
+    type: secret
+    label: Api key
+    help: "Go to your profile and copy your Api key, make sure to replace <+> by <%2B>"
+    regex: /([a-zA-Z0-9\+])
+
+irc:
+  network: p2p-network.net
+  server: irc.p2p-network.net
+  port: 6697
+  tls: true
+  channels:
+    - "#milkie-announce"
+  announcers:
+    - the_cow

```

Comment:
In the `autodl-irssi` .tracker file there's multiple announcers - are any of those used as well or actually only this one?

Reply:
the_cow1 and the_cow2 are not in the irc chan nope, I checked the .tracker file anyway already.

Reply:
Great, thank you for double checking that.